dependencies.gradle

```diff
@@ -17,34 +17,35 @@ ext {
 
     libraries = [
             app : [
-                    autoValue          : 'com.google.auto.value:auto-value:1.5.1',
-                    calligraphy        : 'uk.co.chrisjenx:calligraphy:2.3.0',
```

Comment:
I suppose this is the only change? also why do we keep reformatting every time this change? it makes it harder to understand :)

Reply:
I may have a wrong formatting, let me check

Reply:
I think you do indeed